### ISSUE TITLE
[$20][MSFT-185] Frames must have title attribute

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2853]
**Steps to Reproduce**
1. Navigate to https://www.topcoder.com/challenges/30095347
2. Right click > Inspect > Elements
3. Check
<iframe id="tc-accounts-iframe" src="https://accounts.topcoder.com/connector.html" width="0" height="0"></iframe>

**Expected Result**
<frame> elements should contain a non-empty title attribute

**Actual Result**
Element doesn't have a title attribute

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 4. Robust
**WCAG Level**: A
**WCAG Success Criterion**: 4.1.2 Name, Role, Value
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=37e62274%2D78ab%2D496a%2D918f%2Dabba8f38f3fe&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fyoctocode%2Fbug%207

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095989 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095989 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30095989 has been updated - it has been assigned to nahidshahin.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30095989 has been updated - it has been assigned to nahidshahin.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by nahidshahin** [comment]
@cwdcwd we need 2 PRs https://github.com/appirio-tech/accounts-app/pull/234 & https://github.com/topcoder-platform/community-app/pull/2966

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2853 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2853 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2853 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097094 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097101 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097101 has been updated - it has been assigned to nahidshahin.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097101 has been updated - it has been assigned to nahidshahin.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by veshu** [comment]
> appirio-tech/accounts-app#234

@nahidshahin We can adjust prize, if this fix is working. Thanks!

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097101 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by nkumar-topcoder** [comment]
Dependency testing on community_app, On Hold to be taken once community app issue is merged https://github.com/topcoder-platform/community-app/pull/2966 @sushilshinde 
To be tested jointly via community-app --> account app

**Comment by drasticdpk** [comment]
Unable to see the title attribute., please update accordingly. 
![image](https://user-images.githubusercontent.com/25297859/62005755-9fa25d80-b157-11e9-906e-10558cf4ca0a.png)

**Comment by nkumar-topcoder** [comment]
@mishacucicea  @drasticdpk  I verified for PR merge code for frameTitle and it's there . @sushilshinde can you check for dependency at community-app package.json references "tc-accounts": "git+https://github.com/appirio-tech/accounts-app.git#dev is pointing to `qa-accessiblity` branch ?

**Comment by veshu** [comment]
@sushilshinde FYI

**Comment by nahidshahin** [comment]
@veshu @sushilshinde what is the status of this ticket?

**Comment by crazyk07** [comment]
@tosha5252  Can you check this out too?  Stale for awhile.  Unsure of where we left off, so we can do another dev check and go from there.  May have also been solved by another issue already.

**Comment by tosha5252** [comment]
This is fixed 
![iframe](https://user-images.githubusercontent.com/6400904/62674861-f08d2f80-b969-11e9-8367-1ddbe8c98e7a.png)

**Comment by nahidshahin** [comment]
@crazyk07 @veshu @sunildahiya can you please process the payment?

**Comment by veshu** [comment]
@tosha5252 Is this fixed on production as well?

**Comment by tosha5252** [comment]
Verified on prod 

Win10/Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097101<br/><br/>```This is an automated message for crazyk via Topcoder X```
